templates/css/example.css

```diff
@@ -33,6 +33,8 @@ pre>code {
 .abe-code pre {
   overflow-x: auto;
   white-space: pre;
+  padding: 0;
```

Comment:
unfortunately not - I need to overwrite the more specific styling by ampstart's `www-component-desc .pre` rule.